esmvaltool/recipes/recipe_climwip_test_performance_sigma.yml

```diff
@@ -96,15 +75,24 @@ preprocessors:
       mask_out: sea
     extract_region:
       start_longitude: 0.0
-      end_longitude: 39.0
+      end_longitude: 60.0
       start_latitude: 30.0
-      end_latitude: 76.25
+      end_latitude: 80.0
 
   climatological_mean:
     <<: *general
     climate_statistics:
       operator: mean
 
+  annual_trend:
```

Comment:
Is this what you mean with "tests more different cases"? Could you make the description of the PR a bit more explicit?

Reply:
Yes exactly. I will add a description